.travis.yml

```diff
@@ -1,14 +1,23 @@
+sudo: required
+dist: trusty
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
 language: node_js
 node_js:
-  - "node"
-  - "6"
   - "4"
+  - "6"
 cache:
-  yarn: true
   directories:
     - node_modules
-install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH=$HOME/.yarn/bin:$PATH
-  - yarn --ignore-engines
+before_install:
+  - rm yarn.lock
```

Comment:
Here we install chromedriver with NPM.
It doesn't seems to work properly with yarn + travis-ci's caching setup.

Anyway, moving to NPM back didn't add any considerable amount of time to the build.